### DISCUSSION TITLE
/hyndsight/estimation/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#117] [Announcements]
Different results from different software 
 https://robjhyndman.com/hyndsight/estimation/

**Comment by robjhyndman** [comment]
```header
Originally posted by Juan E. on 2012-07-25 15:05:03
````
Hello Dr. Hyndman,\
You mention that to fit an ARIMA(1,1,0)(1,0,0)4 on Eviews one needs to
use: dlog(fee) c ar(1) sar(4)\
Could you please tell me how to fit an ARIMA(1,1,0)(2,0,0)4 in Eviews?
In particular, since this a multiplicative model, I am not sure about
how to specify the polynomial (1-a4 B\^4 - a8 B\^8) in the Eviews
syntax.

Thank you very much for your time.\
Regards,\
Juan

**Comment by robjhyndman** [comment]
```header
Originally posted by Rachna on 2015-11-02 03:57:28
````
Why Forecasts change when models are fitted using multicore
processing\...please share the insights

**Comment by rrremedio** [comment]
Hello, Dr. Hyndman

A curious case here. 
I followed instructions from this post to try get similar results between softwares. 

I wrote a question on Stackoverflow, but they didn't accept the question saying it was out of their topics. Then I submitted it to R Mailing Lits (https://stat.ethz.ch/pipermail/r-help/2023-January/476615.html) and on reddit (https://www.reddit.com/r/Rlanguage/comments/10evhdo/discrepancies_between_r_and_other_softwares_for/?utm_source=share&utm_medium=android_app&utm_name=androidcss&utm_term=1&utm_content=2). Reddit is better for visualisation of the results. 

Eviews, Stata and Gretl give outputs that differ after a few decimal places, which is expected. R, by its turn, gives completlely different results. All of them used "BFGS" algorithm and maximum liklihood (even eviews). The standard erros, which I'm not questioning here, are very similar between Gretl, Eviews and Stata. In this example, the major difference is in AR(1) coefficient.

As I noted later, according to Eviews documentation, for ARMA models you  have these choices, although they output "nonlinear least squares" in the output screen:

-	For models without fractional differencing, you may choose between the default ML (maximum likelihood), GLS (generalized least squares), and CLS (conditional least squares) estimation.

**Reply by robjhyndman** [reply]
Interesting. Possibly they handle the likelihood differently. R uses a state space model with a diffuse  prior. I doubt that the other software does the same thing -- they may even use a conditional likelihood rather than the full likelihood as it is easier to code.
